skbio/_base.py

```diff
@@ -23,3 +34,381 @@ class SkbioObject(with_metaclass(ABCMeta, object)):
     @abstractmethod
     def __str__(self):
         pass
+
+
+class OrdinationResults(SkbioObject):
+    """Store ordination results, providing serialization and plotting support.
+
+    Stores various components of ordination results. Provides methods for
+    serializing/deserializing results, as well as generation of basic
+    matplotlib 3-D scatterplots. Will automatically display PNG/SVG
+    representations of itself within the IPython Notebook.
+
+    Attributes
+    ----------
+    short_method_name : str
+        Abbreviated ordination method name.
+    long_method_name : str
+        Ordination method name.
+    eigvals : pd.Series
+        The resulting eigenvalues.  The index corresponds to the ordination
+        axis labels
+    samples : pd.DataFrame
+        The position of the samples in the ordination space, row-indexed by the
+        sample id.
+    features : pd.DataFrame
+        The position of the features in the ordination space, row-indexed by
+        the feature id.
+    biplot_scores : pd.DataFrame
+        Correlation coefficients of the samples with respect to the features.
+    sample_constraints : pd.DataFrame
+        A basis for the constrained ordination space. <- JORGE REVISE THIS
```

Comment:
I'd go with a definition from a book. For example, in "Numerical Ecology with R", by Daniel Borcard,Francois Gillet,Pierre Legendre:
> Site constraints (linear combinations of constraining variables): coordinates of the sites in the space of the explanatory variables X. These are the fitted site scores.